pkg/sso/client.go

```diff
@@ -111,6 +112,9 @@ type GetAuthorizationURLOptions struct {
 	// The unique identifier for a WorkOS Connection.
 	Connection string
 
+	//  The organization Id can be used to initiate SSO.
```

Comment:
Could we make this comment analogous to the one for `Connection`?

```suggestion
	//  The unique identifier for a WorkOS Organization.
```